pkg/featureflag/featureflag.go

```diff
@@ -39,7 +40,7 @@ const (
 )
 
 func init() {
-	ParseFlags(os.Getenv(Name))
+	ParseFlags(viper.GetString(Name))
```

Comment:
I think `viper.BindEnv()` would be required here also.
```suggestion
	_ = viper.BindEnv(Name)
	ParseFlags(viper.GetString(Name))
```